client/src/holdings/HoldingsList.tsx

```diff
@@ -168,14 +169,21 @@ export function HoldingsList({user, fxRates}: IProps) {
             <div>Total Performance (incl. sales):</div>
             <PerformanceDisplay performance={totalPerformance} user={user} />
           </div>
-          <Button
-            variant='contained'
-            color='primary'
-            disabled={isRefreshing}
-            onClick={() => refreshPrices()}
-            aria-label='Refresh prices'>
-            Refresh Prices
-          </Button>
+          <Tooltip
+            title={
+              'Last updated at:' +
+              new Date(holdings[0]?.bidPriceUpdateTime).toLocaleString()
+            }
+            arrow>
+            <Button
+              variant='contained'
+              color='primary'
+              disabled={isRefreshing}
+              onClick={() => refreshPrices()}
+              aria-label='Refresh prices'>
+              Refresh Prices
+            </Button>
+          </Tooltip>
```

Comment:
Looks good, didn't we say we were going to hide the button if the user had no holdings though?

Also btw you need to use the word 'fix' in your commit message next to the bug number otherwise github won't pick it up. https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword So it should be:

[Client] Add 'last update time' for prices (fix #37) #47